### ISSUE TITLE
[Feature Request] Collapse search results

### ISSUE THREAD

**Issue by wwjCMP** [TfTHacker/obsidian42-strange-new-worlds#66]
It is great if we can collapse the content of reference. Just like the feature in the official search plugin. Sometimes,  it is more appropriate to show only the location of the reference.

**Comment by VKondakoff** [comment]
Second this.

**Comment by Hydro8** [comment]
Yes it will be a great feature, even more if we can collapse by default !
